### DISCUSSION TITLE
Herramientas CASE

### DISCUSSION THREAD

**Discussion by zherar7ordoya** [community/community#116606] [Discussions] [answered]
### Select Topic Area

Question

### Body

Hola gente. Herramientas CASE. En la universidad usamos Enterprise Architect. Pero existen otras. Hace poco me enteré que el universo Eclipse tiene las suyas (algunas gratuitas y otras de pago). Rational Rose es muy ponderada. Si alguien tuvo la oportunidad de usarlas, ¿podría comentar su experiencia? Gracias.

**Comment by Akash1134** [comment, marked as answer]
Hi @, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
